x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_table/alerts_sub_grouping.tsx

```diff
@@ -215,6 +222,10 @@ export const GroupedSubLevelComponent: React.FC<AlertsTableComponentProps> = ({
     endDate: to,
   });
 
+  const fetchMethod = useMemo(() => {
```

Comment:
This is nit (and also subjective) but what do you think about writing this instead?
```typescript
const fetchMethod = useMemo(() =>
  pageScope === PageScope.attacks ? fetchQueryUnifiedAlerts : fetchQueryAlerts
);
```
I know `fetchQueryAlerts` is set by default  for the `fetchMethod`, but I think it makes this piece of code much easier to read, insead of having `undefined` here which is a bit confusing.